wikimedia/wikipedia-ios
# Remove Tweaks from additional targets

There were a couple of other references to Tweaks in the build settings that caused the Experimental build to fail. This fixes that.

Test steps:

Choose the Experimental target, select Product > Archive. Confirm it archives without failure.
